.docker_utils/main_dock.sh

```
@@ -1,5 +1,12 @@
 #! /bin/bash
 
+# Script to create a Xauthority file for the docker container - copied from xauth_setup.sh to avoid issues
+XAUTH=$HOME/.Xauthority
+export XAUTH_LOC=$XAUTH
+xhost +local:$USER
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)